tests/integration/frameworks/wordpress/test_wordpress_do_action.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should properly instrument Wordpress do_action hooks.
+*/
+
+/*SKIPIF
+<?php
+if (version_compare(PHP_VERSION, "5.6", "<")) {
+  die("skip: PHP < 5.6 argument unpacking not supported\n");
+}
```

Comment:
Since we don't build/support PHP < 7.0 we should be able to remove this SKIPIF entirely

Reply:
Addressed with b9fb84d9c223bd4548c065e62a1cba4d8bdbe235.